src/mito2/src/region.rs

```diff
@@ -38,6 +38,22 @@ use crate::sst::file_purger::FilePurgerRef;
 /// This is the approximate factor to estimate the size of wal.
 const ESTIMATED_WAL_FACTOR: f32 = 0.42825;
 
+/// Region status include region id, memtable usage, sst usage, wal usage and manifest usage.
+#[derive(Debug)]
+pub struct RegionStat {
+    pub region_id: RegionId,
+    pub memtable_usage: u64,
+    pub wal_usage: u64,
+    pub sst_usage: u64,
+    pub manifest_usage: u64,
+}
+
+impl RegionStat {
+    pub fn total_usage(&self) -> u64 {
+        self.memtable_usage + self.wal_usage + self.sst_usage + self.manifest_usage
```

Comment:
What we need is disk usage so it should not include memtable usage.
```suggestion
        self.wal_usage + self.sst_usage + self.manifest_usage
```